lambda/fulfillment/lib/middleware/1_parse.js

```diff
@@ -102,6 +102,9 @@ function getClientType(req) {
     } else if (_.get(req,"_event.requestAttributes.x-amz-lex:accept-content-types")) {
         return "LEX.AmazonConnect." + voiceortext ;
     }
+    else if (_.get(req,"_event.requestAttributes.x-amz-lex:channels:platform") == "Genesys Cloud") {
```

Comment:
Since Genesys Cloud support was added to Lex V2, is there a need to handle miss-configuration if one has miss-configured the solution with Lex V1 but get requests from Gensys Cloud? What should happen in that case?

Reply:
LexV2 always gets deployed, and Genesys will always reference LexV2 version of the bot, so this is non issue.